index.js

```diff
@@ -1,17 +1,35 @@
 
-var { benchBranch, benchmarkRuntime } = require("./bench");
+const { benchBranch, benchmarkRuntime } = require("./bench");
+
+let allowedUsers = process.env.ALLOWED_USERS || "";
```

Comment:
this is list of usernames or github ids?

Reply:
github ids.

Reply:
it should be specified somewhere

Reply:
I will add it to readme.